src/Sentry.Unity.Editor/ConfigurationWindow/SentryEditorWindowInstrumentation.cs

```diff
@@ -73,6 +73,18 @@ private static void ConfigureOptions(Dictionary<string, string> args, [CallerMem
                 options.Il2CppLineNumberSupportEnabled = boolValue;
             }
 
+            if (args.TryGetValue("traceSampleRate", out boolValue))
+            {
+                Debug.LogFormat("{0}: Configuring TraceSampleRate to {1}", functionName, boolValue ? 1.0f : 0.0f);
+                options.TracesSampleRate = boolValue ? 1.0f : 0.0f;
```

Comment:
Use `d` instead of `f`, since `options.TracesSampleRate` is a `double`, not a `float`.